src/js/controller/controller.js

```diff
@@ -440,6 +440,9 @@ Object.assign(Controller.prototype, {
         function _checkPlayOnViewable(model, viewable) {
             const adState = _getAdState();
             if (model.get('playOnViewable')) {
+                if (adState) {
+                    _this._instreamAdapter.noResume = !viewable;
+                }
```

Comment:
Shouldn't you use `_pauseAfterAd`? It's called below, but only on mobile. So if you deleted `OS.mobile && ` below isn't that the same as adding this code?
```
                if (OS.mobile && adState) {
                    // If during an ad on mobile, we should always be paused after the ad,
                    // if not viewable, regardless of autoPause setting.
                    _pauseAfterAd(viewable);
                }
```

Reply:
That updates the `pauseReason` and causes the playback to resume on viewable even if the user interacts to pause the player